File: uploader/api/models/data_unit.py
class DataUnit():
    def __init__(self, period, measurement, timestamp):
        self.period = period
        self.measurement = measurement
        self.timestamp = timestamp
    
    def asDict(self):
        return {
            'period': self.period,
            'measurement': self.measurement,
            'timestamp': self.timestamp,
        }

File: backend/modules/uploader/api/models/dataUnit.js
class DataUnit {
    constructor(period, measurement, timestamp) {
        this.period = period;
        this.measurement = measurement;
        this.timestamp = timestamp;
    }

    asObject() {
        return {
            period: parseInt(this.period),
            measurement: parseFloat(this.measurement),
            timestamp: this.timestamp,
        };
    }
}

module.exports = {
    DataUnit,
};